README.md

```diff
@@ -47,6 +47,12 @@ Install [Postgres](https://www.postgresql.org/download/) (on OS X you can use `b
 CREATE USER feedadoc WITH SUPERUSER PASSWORD 'password1';
 ```
 
+2. Set the database url manually or add this command to a file like `.bashrc`:
+
+```
+export DATABASE_URL="postgresql://localhost"
```

Comment:
This is documented in the Docker section. Maybe add a comment [there](https://github.com/feedadoc/feedadoc/pull/88/files#diff-04c6e90faac2675aa89e2176d2eec7d8R38)?

Reply:
Or, you could add a `## Troubleshooting` section.

The third option is that we suggest that users make a `.env` file and we add [dotenv](https://github.com/bkeepers/dotenv) to the Gemfile. Do you have a preference?

Reply:
Oh my bad, I didn't try running the command in the Docker section. I don't really have a preference. 

If the steps are the same, then how about we tell them to refer to step 2 in the Docker section after completing step 1?

If you think that dotenv is suitable, then I can add that, too.

Reply:
Were you using Docker, or just running this locally with a local postgres process?

Reply:
I wasn't using Docker, just ran with a local DB.